XCResultViewer.xcodeproj/project.pbxproj

```diff
@@ -326,7 +326,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				CODE_SIGN_STYLE = Manual;
-				DEVELOPMENT_TEAM = 9BCL7VAFF4;
+				DEVELOPMENT_TEAM = VG9R2N57GC;
```

Comment:
And this one :P also the xcuserdatad below 🙈

Reply:
Hmm I've set the Development Team to None, hopefully that fixes it.

Reply:
I mean that's going to mean I have to change it back to mine to be able to export it no?